zuul.d/project-templates.yaml

```diff
@@ -1392,6 +1392,25 @@
               - name: release-ansible-collection-galaxy
                 soft: true
 
+- project-template:
+    name: publish-to-automation-hub-3pci
+    description: |
+      Publish an Ansible collection to Ansible Automation Hub
+    third-party-check:
```

Comment:
this is the wrong pipeline for 3pci-check

Reply:
oh indeed. Let's fix that.

Reply:
https://github.com/ansible/ansible-zuul-jobs/pull/1124